win32ss/user/user32_vista/CMakeLists.txt

```
@@ -0,0 +1,25 @@
+
+spec2def(user32_vista.dll user32_vista.spec ADD_IMPORTLIB)
+
+include_directories(
+    include
+    ${REACTOS_SOURCE_DIR}/win32ss/include)
+
+list(APPEND SOURCE
+    dpi.c)
+
+add_library(user32_vista_static
+    ${SOURCE})
+target_link_libraries(user32_vista_static win32ksys)
+add_dependencies(user32_vista_static psdk)
+
+add_library(user32_vista MODULE
+    ${SOURCE}
+    user32_vista.rc
+    ${CMAKE_CURRENT_BINARY_DIR}/user32_vista.def)
+
+set_module_type(user32_vista win32dll UNICODE ENTRYPOINT 0)
+add_importlibs(user32_vista user32 gdi32)
+target_link_libraries(user32_vista win32ksys)
```

Should this dll use the static lib (instead)?
```suggestion
target_link_libraries(user32_vista user32_vista_static win32ksys)
```

---

No, if I'm understanding you correctly. The purpose of `user32_vista_static` is to allow `user32` to use function definitions from `user32_vista`.

---

'kernel32*' seem to use it for both purposes:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=_vista_static

---

Added.

---

> Added.

Related `${SOURCE}` should be removed, shouldn't it?